### PR TITLE
User documents restructuring (2,3/3)

### LINKED ISSUE #10066: Improve `control-plane-endpoints-and-ports.md`
**How to categorize this issue?**

/area documentation
/kind enhancement

**What would you like to be added**:
The following sentences need change:
1. "The Kubernetes client sets the server name to `api.<external-domain>` or `api.<shoot>.<project>.<internal-domain>`." can be dropped because all modern clients do this thing.
2. "Details can be found in [GEP-11](https://github.com/gardener/gardener/blob/master/docs/proposals/11-apiserver-network-proxy.md)." GEP-11 is dropped and here GEP-8 was meant

Also the following document is under the users section but it mostly contains implementation logic not relevant for users. The only user relevant information is the list of the opened ports. This information is relevant only when a user wants to configure a firewall and needs to allow these ports.

Therefore we can move this document to be under the developer persona and create a "Configuring a firewall" document where the ports will be listed. And if more information is needed it will link to the moved `control-plane-endpoints-and-ports.md`

**Why is this needed**:

### PR DESCRIPTION
**Pull request by Kostov6**
**How to categorize this PR?**

/area documentation
/kind enhancement

**What this PR does / why we need it**:
This PR contains the final parts from https://github.com/gardener/gardener/pull/9799 and is a follow up on https://github.com/gardener/gardener/pull/10077 that added titles and descriptions to user documents. The logic behind the changes in this PR are well described in https://github.com/gardener/gardener/pull/9799

**Which issue(s) this PR fixes**:
Part of https://github.com/gardener/documentation/issues/488
Fixes https://github.com/gardener/gardener/issues/10066

**Special notes for your reviewer**:
I tried making the commit messages reflect the actions done as best as possible

**Release note**:

```doc user
Majority of user documents are in a dedicated section and irrelevant documents are moved out
```

### CODE REVIEW

File: docs/usage/shoot_basics/basic-configuration.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: Basic Shoot Configurations
+weight: 6
+---
+
+The [gardener dashboard](https://github.com/gardener/dashboard/blob/master/README.md) provides a simple way for creating and configuring a basic shoot cluster. You can see how changing the shoot configuration through the overview section reflects on the shoot yaml resource. When talking about clusters in the documentation the yaml structure will be mostly refered and therefore a basic understanding of it's structure will be highly usefull.
```
Comment (n-boshnakov) [inline docs/usage/shoot_basics/basic-configuration.md]:
```suggestion
The [gardener dashboard](https://github.com/gardener/dashboard/blob/master/README.md) provides a simple way for creating and configuring a basic shoot cluster. You can see how changing the shoot configuration through the overview section reflects on the shoot yaml resource. When talking about clusters in the documentation, the yaml structure will be referred to, and therefore a basic understanding of its structure will be highly useful.
```

File: docs/usage/shoot_basics/creating-shoot.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+title: Creating a Shoot Cluster
+weight: 4
+---
+
+You can check the tutorial section for the different providers in [Infrastructure Extensions](https://gardener.cloud/docs/extensions/infrastructure-extensions/)
```
Comment (n-boshnakov) [inline docs/usage/shoot_basics/creating-shoot.md]:
```suggestion
You can check the tutorial section for the different providers in [Infrastructure Extensions](https://gardener.cloud/docs/extensions/infrastructure-extensions/)
```
```suggestion
You can check the tutorial section for the different providers in the [Infrastructure Extensions](https://gardener.cloud/docs/extensions/infrastructure-extensions/) section.
```

File: docs/usage/shoot_firewall.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+Depending on the configured zones or [exposure classes](./nessesary_considerations/exposureclasses.md), there are different endpoints in a seed cluster. The IP address(es) can be determined by a DNS query for the API Server URL.
+The main entry-point into the seed cluster is the load balancer of the Istio ingress-gateway service. Depending on the infrastructure provider, there can be one IP address per zone.
+
+The load balancer of the Istio ingress-gateway service exposes the following TCP ports:
+
+* **443** for requests to the shoot API Server. The request is dispatched according to the set TLS SNI extension.
+* **8443** for requests to the shoot API Server via `api-server-proxy`, dispatched based on the proxy protocol target, which is the IP address of `kubernetes.default.svc.cluster.local` in the shoot.
+* **8132** to establish the reversed VPN connection. It's dispatched according to an HTTP header value.
+
+For detailed information you can check [Control Plane Endpoints and Ports](../development/control-plane-endpoints-and-ports.md)
```
Comment (n-boshnakov) [inline docs/usage/shoot_firewall.md]:
```suggestion
For detailed information, you can check [Control Plane Endpoints and Ports](../development/control-plane-endpoints-and-ports.md).
```

File: docs/usage/shoot_settings/default_seccomp_profile.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+---
+description: Enable the use of `RuntimeDefault` as the default seccomp profile through `spec.kubernetes.kubelet.seccompDefault`
+weight: 15
+---
+
+# Default Seccomp Profile and Configuration 
+
+This is a short guide describing how to enable the defaulting of seccomp profiles for shoot clusters. Running pods in `Unconfined` (seccomp disabled) mode is undesirable since this is the least restrictive profile. Also, mind that any privileged container will always run as `Unconfined`. More information about seccomp can be found in this [Kubernetes tutorial](https://kubernetes.io/docs/tutorials/security/seccomp/).
```
Comment (n-boshnakov) [inline docs/usage/shoot_settings/default_seccomp_profile.md]:
```suggestion
This is a short guide describing how to enable the defaulting of seccomp profiles for shoot clusters. Running pods in `Unconfined` (seccomp disabled) mode is undesirable since this is the least restrictive profile. Also, mind that any privileged container will always run as `Unconfined`. More information about seccomp can be found in [Restrict a Container's Syscalls with seccomp](https://kubernetes.io/docs/tutorials/security/seccomp/).
```

File: docs/usage/shoot_settings/shoot_autoscaling.md [hunk 5]
```diff
@@ -21,21 +22,21 @@ Consequently, please refer to the [official documentation](https://github.com/ku
...
 > Only some `cluster-autoscaler` flags can be configured per worker pool, and is limited by NodeGroupAutoscalingOptions of the upstream community Kubernetes repository. This list can be found [here](https://github.com/gardener/autoscaler/blob/machine-controller-manager-provider/cluster-autoscaler/config/autoscaling_options.go#L37-L55).
 
 ## Vertical Pod Auto-Scaling
 
 This form of auto-scaling is not enabled by default and must be explicitly enabled in the `Shoot` by setting `.spec.kubernetes.verticalPodAutoscaler.enabled=true`.
-The reason is that it was only introduced lately, and some end-users might have already deployed their own VPA into their clusters, i.e., enabling it by default would interfere with such custom deployments and lead to issues, eventually.
+The reason is that it was only introduced lately, and some end-users might have already deployed their own VPA into their clusters, i.e., enabling it by default would interfere with such custom deployments and lead to issues, eventually. Also, if `ShootVPAEnabledByDefault` admissionPlugins is enabled, then `.spec.kubernetes.verticalPodAutoscaler.enabled` will be set to true.
```
Comment (n-boshnakov) [inline docs/usage/shoot_settings/shoot_autoscaling.md]:
```suggestion
The reason is that it was only introduced lately, and some end-users might have already deployed their own VPA into their clusters, i.e., enabling it by default would interfere with such custom deployments and lead to issues, eventually. Also, if the `ShootVPAEnabledByDefault` admissionPlugins is enabled, then `.spec.kubernetes.verticalPodAutoscaler.enabled` will be set to true.
```

**Review by n-boshnakov — CHANGES_REQUESTED** [review]

File: docs/README.md [hunk 6]
```diff
@@ -25,52 +25,48 @@
...
+* [Audit a Kubernetes cluster](usage/nessesary_considerations/shoot_auditpolicy.md)
+* [Auto-Scaling for shoot clusters](usage/shoot_settings/shoot_autoscaling.md)
+* [Cleanup of Shoot clusters in deletion](usage/nessesary_considerations/shoot_cleanup.md)
+* [`containerd` Registry Configuration](usage/containerd/containerd-registry-configuration.md)
+* [Custom `containerd` configuration](usage/containerd/custom-containerd-config.md)
+* [Custom `CoreDNS` configuration](usage/dns/custom-dns-config.md)
+* [(Custom) CSI components](usage/nessesary_considerations/csi_components.md)
+* [Default Seccomp Profile](usage/shoot_settings/default_seccomp_profile.md)
+* [DNS Autoscaling](usage/dns/dns-autoscaling.md)
+* [DNS Search Path Optimization](usage/dns/dns-search-path-optimization.md)
```
Comment (ialidzhikov) [inline docs/README.md]: As you now introduce different sections, can we also have the links grouped by sections and by weights and not alphabetically. You aim to fix the poor alphabetic order for the website but we could also fix it in this document.

```

## Usage

### Shoot Basics

- ...
-...

### Configuring the Shoot Spec

- ...
- ...

### ...

- ...
```

File: docs/usage/containerd/containerd-registry-configuration.md [hunk 7]
```diff
@@ -1,5 +1,6 @@
 ---
 title: containerd Registry Configuration
+weight: 2
```
Comment (ialidzhikov) [inline docs/usage/containerd/containerd-registry-configuration.md:3]: Can you elaborate based on what the weights are set and what is their purpose? When I introduce a new document tomorrow based on what I will decide on the weight? This has to be documented as convention somewhere in the developer docs.

File: docs/README.md [hunk 8]
```diff
@@ -25,52 +25,48 @@
 
 ## Usage
 
-* [Audit a Kubernetes cluster](usage/shoot_auditpolicy.md)
```
Comment (ialidzhikov) [inline docs/README.md:28]: As suggested in https://github.com/gardener/gardener/pull/9799#discussion_r1660551278, use imperative for the commit messages. Use `Move documents under usage, add weights and fix links` instead of `Documents under usage moving, adding weight and fixing links`.

File: docs/usage/logging.md [hunk 9]
```diff
@@ -1,3 +1,7 @@
+---
```
Comment (ialidzhikov) [inline docs/usage/logging.md:1]: We have to find sections for all docs. Currently there are 6 doc files that live in the usage dir's root:
```
% ls docs/usage/ | grep md
logging.md
openidconnect-presets.md
projects.md
service-account-manager.md
shoot_firewall.md
shoot_high_availability_best_practices.md
```

File: docs/README.md [hunk 10]
```diff
@@ -25,52 +25,48 @@
...
-* [Auto-Scaling for shoot clusters](usage/shoot_autoscaling.md)
-* [Cleanup of Shoot clusters in deletion](usage/shoot_cleanup.md)
-* [`containerd` Registry Configuration](usage/containerd-registry-configuration.md)
-* [Custom `containerd` configuration](usage/custom-containerd-config.md)
-* [Custom `CoreDNS` configuration](usage/custom-dns-config.md)
-* [(Custom) CSI components](usage/csi_components.md)
-* [Default Seccomp Profile](usage/default_seccomp_profile.md)
-* [DNS Autoscaling](usage/dns-autoscaling.md)
-* [DNS Search Path Optimization](usage/dns-search-path-optimization.md)
+* [Audit a Kubernetes cluster](usage/nessesary_considerations/shoot_auditpolicy.md)
```
Comment (ialidzhikov) [inline docs/README.md:28]: `Considerations for Specific Scenarios` is not a good section name, IMO. Something like `Advanced use cases` is better IMO.

File: docs/README.md [hunk 11]
```diff
@@ -25,52 +25,48 @@
...
-* [Trusted TLS certificate for garden runtime cluster](usage/trusted-tls-for-garden-runtime.md)
-* [Controlling the Kubernetes versions for specific worker pools](usage/worker_pool_k8s_versions.md)
-* [Admission Configuration for the `PodSecurity` Admission Plugin](usage/pod-security.md)
-* [Supported CPU Architectures for Shoot Worker Nodes](usage/shoot_supported_architectures.md)
-* [Workerless `Shoot`s](usage/shoot_workerless.md)
+* [Shoot Workers Settings](usage/shoot_settings/shoot_workers_settings.md)
+* [Accessing Shoot Clusters](usage/shoot_basics/shoot_access.md)
+* [Supported Kubernetes versions](usage/shoot_updates_and_upgrades/supported_k8s_versions.md)
+* [Tolerations](usage/nessesary_considerations/tolerations.md)
+* [Trigger shoot operations](usage/operating_through_annotations/shoot_operations.md)
```
Comment (ialidzhikov) [inline docs/README.md:65]: Instead of `Operating via Shoot Annotations` I would use as a section name `Shoot Operations`. It is implementation detail that the operations are triggered by well-known annotations. Tomorrow there can be a case where an operation is triggered differently.

File: docs/usage/nessesary_considerations/csi_components.md [hunk 12]
```diff
@@ -24,5 +25,4 @@ Custom CSI components are typically regular `Deployment`s running in the shoot c
 
 When a shoot cluster is deleted, Gardener deletes most Kubernetes resources (`Deployment`s, `DaemonSet`s, `StatefulSet`s, etc.). Gardener will also try to delete CSI components if they are not marked with the above mentioned label.
 
-This can result in `VolumeSnapshot` resources still having finalizers that will never be cleaned up.
-Consequently, manual intervention is required to clean them up before the cluster deletion can continue.
+This can result in `VolumeSnapshot` resources leaking in the cloud providers and manual cleanup will be required.
```
Comment (ialidzhikov) [inline docs/usage/nessesary_considerations/csi_components.md:28]: Good point, we started to finalize VolumeSnapshot resources and they will no longer block Shoot deletion.

File: docs/usage/shoot_basics/shoot_status.md [hunk 13]
```diff
@@ -1,14 +1,15 @@
...
 
 # Shoot Status
 
-This document provides an overview of the [ShootStatus](../api-reference/core.md#shootstatus).
+This document provides an overview of the [ShootStatus](../../api-reference/core.md#shootstatus).
 
 ## Conditions
 
-The Shoot status consists of a set of conditions. A [Condition](../api-reference/core.md#condition) has the following fields:
+The Shoot status consists of a set of conditions. They represent the latest available observations of a Shoot’s current state. A [Condition](../../api-reference/core.md#condition) has the following fields:
```
Comment (ialidzhikov) [inline docs/usage/shoot_basics/shoot_status.md:12]: I am not sure if there is a need for `current`.

```suggestion
The Shoot status consists of a set of conditions. They represent the latest available observations of the Shoot’s state. A [Condition](../../api-reference/core.md#condition) has the following fields:
```

File: docs/usage/shoot_settings/shoot_autoscaling.md (hunk 5 above)
Comment (ialidzhikov) [inline docs/usage/shoot_settings/shoot_autoscaling.md]:
```suggestion
The reason is that it was only introduced lately, and some end-users might have already deployed their own VPA into their clusters, i.e., enabling it by default would interfere with such custom deployments and lead to issues, eventually. Additionally, if the `ShootVPAEnabledByDefault` admission plugin is enabled in the gardener-apiserver, then the `.spec.kubernetes.verticalPodAutoscaler.enabled` field will be defaulted to true.
```

Add also a link to the admission plugin docs in `apiserver-admission-plugins.md`.

File: docs/usage/shoot_settings/shoot_hibernate.md [hunk 14]
```diff
@@ -27,22 +20,22 @@ When a cluster is hibernated, Gardener scales down the worker nodes and the clus
...
+### What Isn’t Affected by the Hibernation?
 
 To scale up everything where it was before hibernation, Gardener doesn’t delete state-related information, that is, information stored in persistent volumes. The cluster state as persistent in `etcd` is also preserved.
 
 ## Hibernate Your Cluster Manually
 
 The `.spec.hibernation.enabled` field specifies whether the cluster needs to be hibernated or not. If the field is set to `true`, the cluster's desired state is to be hibernated. If it is set to `false` or not specified at all, the cluster's desired state is to be awakened.
 
-To hibernate your cluster, you can run the following `kubectl` command:
+To hibernate your cluster, you can do it from the Gardener dashboard or run the following `kubectl` command:
```
Comment (ialidzhikov) [inline docs/usage/shoot_settings/shoot_hibernate.md:31]: I would revert the Gardener dashboard part, we don't have to remind every time that every operation is possible from the dashboard as well.

File: docs/README.md [hunk 15]
```diff
@@ -25,52 +25,47 @@
...
+* [Shoot Updates and Upgrades](usage/shoot_updates_and_upgrades/shoot_updates.md)
+* [Shoot HA Control Plane](usage/shoot_settings/shoot_high_availability.md)
 * [Shoot HA Best Practices](usage/shoot_high_availability_best_practices.md)
-* [Shoot Workers Settings](usage/shoot_workers_settings.md)
-* [Accessing Shoot Clusters](usage/shoot_access.md)
-* [Supported Kubernetes versions](usage/supported_k8s_versions.md)
-* [Tolerations](usage/tolerations.md)
-* [Trigger shoot operations](usage/shoot_operations.md)
-* [Trusted TLS certificate for shoot control planes](usage/trusted-tls-for-control-planes.md)
-* [Trusted TLS certificate for garden runtime cluster](usage/trusted-tls-for-garden-runtime.md)
```
Comment (ialidzhikov) [inline docs/README.md:70]: Move these links to the operations section, don't remove them. The same applies to all documents that were moved from `usage/` to `operations/`.

File: docs/usage/shoot_basics/basic-configuration.md (hunk 1 above)
Comment (ialidzhikov) [inline docs/usage/shoot_basics/basic-configuration.md]: https://github.com/gardener/gardener/pull/9799#discussion_r1660570896

Apply all the comments posted in the old PR: https://github.com/gardener/gardener/pull/9799

File: docs/operations/default_seccomp_profile_seed.md [hunk 16]
```diff
@@ -1,5 +1,5 @@
 ---
-description: Enable the use of `RuntimeDefault` as the default seccomp profile through `spec.kubernetes.kubelet.seccompDefault`
+title: Default Seccomp Profile for Seeds
```
Comment (ialidzhikov) [inline docs/operations/default_seccomp_profile_seed.md:2]: In https://github.com/gardener/gardener/pull/9799#pullrequestreview-2147430906 I suggested 3 PRs where the 2nd PR contains only doc changes. Now a reviewer has to pay attention to multiple things at the same time - content changes and to section changes.

File: docs/operations/default_seccomp_profile_seed.md [hunk 17]
```diff
@@ -32,25 +32,3 @@ featureGates:
 Please refer to the examples in this [yaml file](../../example/20-componentconfig-gardenlet.yaml) for more information.
 
 Once the feature gate is enabled, the webhook will be registered and configured for the seed cluster. Newly created pods will be mutated to have their seccomp profile set to `RuntimeDefault`.
-
-> **Note:** Please note that this feature is still in Alpha, so you might see instabilities every now and then.
```
Comment (ialidzhikov) [inline docs/operations/default_seccomp_profile_seed.md:36]: Why is this sentence removed? To my knowledge the `DefaultSeccompProfile` feature gate of gardenlet is still alpha.

File: docs/usage/shoot_settings/shoot_networking.md [hunk 18]
```diff
@@ -9,9 +10,6 @@ This document contains network related information for Shoot clusters.
 
 ## Pod Network
 
-A Pod network is imperative for any kind of cluster communication with Pods not started within the Node's host network.
```
Comment (ialidzhikov) [inline docs/usage/shoot_settings/shoot_networking.md:12]: What is the motivation for the changes in this file and what we improve with them?

File: docs/usage/shoot_firewall.md [hunk 19]
```diff
@@ -0,0 +1,19 @@
+---
+title: Shoot Firewall Configuration
+weight: 15
+---
+
+# Endpoints and Ports of a Shoot Control-Plane
```
Comment (ialidzhikov) [inline docs/usage/shoot_firewall.md:6]: Why the title is different from the header in the markdown?

File: docs/usage/shoot_firewall.md [hunk 20]
```diff
@@ -0,0 +1,19 @@
...
+
+With the [reversed VPN](../operations/reversed-vpn-tunnel.md) tunnel, there are no endpoints with open ports in the shoot cluster required by Gardener.
+In order to allow communication to the shoots control-plane in the seed cluster, there are endpoints shared by multiple shoots of a seed cluster.
```
Comment (ialidzhikov) [inline docs/usage/shoot_firewall.md:9]: Do you mean that all Shoots on a Seed use the istio ingress-gateway load balancer? Then we should rather rework like "there are endpoints shared by all? Shoots running on a Seed cluster".

File: docs/usage/shoot_firewall.md [hunk 21]
```diff
@@ -0,0 +1,19 @@
...
+Depending on the configured zones or [exposure classes](./nessesary_considerations/exposureclasses.md), there are different endpoints in a seed cluster. The IP address(es) can be determined by a DNS query for the API Server URL.
+The main entry-point into the seed cluster is the load balancer of the Istio ingress-gateway service. Depending on the infrastructure provider, there can be one IP address per zone.
```
Comment (ialidzhikov) [inline docs/usage/shoot_firewall.md:11]: > Depending on the infrastructure provider, there can be one IP address per zone.

Is it really depending on the infrastructure provider or how the istio is configured in the Seed spec?

File: docs/usage/shoot_settings/default_seccomp_profile.md [hunk 22]
```diff
@@ -5,7 +5,7 @@ weight: 15
 
 # Default Seccomp Profile and Configuration 
 
-This is a short guide describing how to enable the defaulting of seccomp profiles for shoot clusters. Running pods in `Unconfined` (seccomp disabled) mode is undesirable since this is the least restrictive profile. Also, mind that any privileged container will always run as `Unconfined`. More information about seccomp can be found in this [Kubernetes tutorial](https://kubernetes.io/docs/tutorials/security/seccomp/).
+This is a short guide describing how to enable the defaulting of seccomp profiles for shoot clusters. Running pods in `Unconfined` (seccomp disabled) mode is undesirable since this is the least restrictive profile. Also, mind that any privileged container will always run as `Unconfined`. More information about seccomp can be found in [Restrict a Container's Syscalls with seccomp](https://kubernetes.io/docs/tutorials/security/seccomp/).
```
Comment (ialidzhikov) [inline docs/usage/shoot_settings/default_seccomp_profile.md:8]: You could apply the same diff to `docs/operations/default_seccomp_profile_seed.md` as the section is the same for both of the files.

**Review by ialidzhikov — COMMENTED** [review]
As shared inline and as initially suggested, it would be more appreciated if the doc changes unrelated to the section changes were split in another PR. But anyways, let's proceed with this PR.

File: docs/usage/containerd/containerd-registry-configuration.md (hunk 7 above)
Comment (Kostov6) [reply docs/usage/containerd/containerd-registry-configuration.md:3]: From the closed PR:
- adding meaningful order of pages. My approach was to make it based on content simplicity and expected user benefit from the content (simpler content and more user benefit push a given page to the top) with simplicity having more weight than expected benefit. The reason behind this is that "user benefit" has high variance (different users benefit differently from different content) and is more subjective than content simplicity which is more objective. This ordering heuristic makes the order more stable while pushing more beneficial pages to the top

I agree that having it documented in a developer docs would be greatly beneficial

File: docs/usage/logging.md (hunk 9 above)
Comment (Kostov6) [reply docs/usage/logging.md:1]: I agree that it's best for every page to have its designated section. I just couldn't find a fitting section for those pages and left them as is

File: docs/README.md (hunk 10 above)
Comment (Kostov6) [reply docs/README.md:28]: I agree

File: docs/operations/default_seccomp_profile_seed.md (hunk 17 above)
Comment (Kostov6) [reply docs/operations/default_seccomp_profile_seed.md:36]: It was removed after asking @dimityrmirchev about it's relevance

Comment (dimityrmirchev) [reply docs/operations/default_seccomp_profile_seed.md:36]: Yes, the feature is in alpha but pods will be mutated reliably with `RuntimeDefault` seccomp profile. I am not sure that there would be any instabilities w.r.t. the feature being `Alpha`.

File: docs/usage/shoot_settings/shoot_networking.md (hunk 18 above)
Comment (Kostov6) [reply docs/usage/shoot_settings/shoot_networking.md:12]: The text explanation and the examples to be grouped together. Also the above sentence was removed because it does not provide any value

File: docs/usage/shoot_firewall.md (hunk 20 above)
Comment (Kostov6) [reply docs/usage/shoot_firewall.md:9]: This was copy-pasted from `Endpoints and Ports of a Shoot Control-Plane`

File: docs/usage/shoot_firewall.md (hunk 21 above)
Comment (Kostov6) [reply docs/usage/shoot_firewall.md:11]: This was copy-pasted from `Endpoints and Ports of a Shoot Control-Plane`

File: docs/usage/shoot_firewall.md (hunk 19 above)
Comment (Kostov6) [reply docs/usage/shoot_firewall.md:6]:
```
# Shoot firewall

## Endpoints and Ports of a Shoot Control-Plane
```
Makes more sense

**Comment by Kostov6** [comment]
Closing as a PR with better categorization will be added instead
